10/root/docker-entrypoint.sh

```diff
@@ -38,4 +38,26 @@ postmap /etc/postfix /etc/postfix/virtual_aliases
 newaliases
 postfix start
 
-exec "$@"
+_term() {
+  echo -n "Stopping postfix..."
+  postfix stop
+  echo "OK"
+  echo -n "Stopping rsyslog..."
+  service rsyslog stop
+  echo "OK"
+  export EXIT=1
+}
+trap _term SIGTERM
+
+# send log to stdout
+tail -F /var/log/mail.log &
+# send error to stderr
+>&2 tail -F /var/log/mail.err &
```

Comment:
The file mail.err does not seem to exist yet, so the tail command fail.

Reply:
With `tail -F` it read mail.err as soon as it is created by postfix :

```
postfix_1  | tail: cannot open '/var/log/mail.err' for reading: No such file or directory
...
postfix_1  | tail: '/var/log/mail.err' has appeared;  following new file
postfix_1  | Jan  3 12:51:54 9c5fdf38efca postfix/smtp[194]: fatal: unknown service: smtp/tcp
```
The last line about "unknown service: smtp/tcp" came from `mail.err` file
You can test it with : 
```bash
make build
make clean
make up
make test
```